openshift-ci/Dockerfile

```diff
@@ -31,11 +31,11 @@ RUN microdnf install -y git zip unzip &&\
 	[ ! -d /var/cache/yum ] || rm -rf /var/cache/yum
 
 # these versions should be updated for every release
-ENV CAMEL_QUARKUS_TS_BRANCH=3.2.x
-ENV QUARKUS_VERSION=3.2.10.Final-redhat-00002
+ENV CAMEL_QUARKUS_TS_BRANCH=main
```

Comment:
@jirisimek98 Can you please create `3.8.x` branch in CQ test-suite from `main` and use it here ? `main` should be always pointing to latest version (now still uses 3.8.x) but in near future it will be used for 3.12 (and some breaking changes could break this interop testing.

Reply:
Done

Reply:
Done

Reply:
Wait, no. Not done